Description: Replace rle-decode-fast with std provided function
 as suggested by that crate itself.
Last-Update: 2024-10-13
--- a/src/lib.rs
+++ b/src/lib.rs
@@ -12,3 +12,2 @@
 use core2::io;
-use rle_decode_fast::rle_decode;
 
@@ -188,7 +187,8 @@
                 }
-                rle_decode(
-                    &mut self.buffer,
-                    usize::from(backward_distance),
-                    usize::from(length),
-                );
+				let old_len = self.buffer.len();
+				let target_len = old_len + length as usize;
+				while self.buffer.len() < target_len {
+					self.buffer.extend_from_within(old_len - backward_distance as usize..old_len);
+				}
+				self.buffer.truncate(target_len);
             }
--- a/Cargo.toml
+++ b/Cargo.toml
@@ -32,4 +32,2 @@
 
-[dependencies.rle-decode-fast]
-version = "1.0.0"
 
